app/javascript/components/admin/server_rooms/ServerRooms.jsx

```diff
@@ -58,15 +58,18 @@ export default function ServerRooms() {
                           )}
                       </tbody>
                     </Table>
+                    {!isLoading
+                      && (
+                        <div className="table-pagination-wrapper">
```

Comment:
At this step of development, Let's start considering using `react-bootstrap` components instead of raw html elements whenever possible,
A `Container` will render a div by default with the supervision of our design lib.

So, `<Container className="table-pagination-wrapper"> ... </Container>` will do the trick,

Reply:
I disagree with this.

I want only the wrapping class, not a container.

Stacking layers of containers - or whatever Bootstrap layout classes - will cause problems (extra paddings, margins, etc), especially with responsiveness.

Reply:
The `Container` will only have `container` class as I recall, the `Row` component will have the `row` class.

For responsiveness I believe it's quite the opposite, when using raw `div` elements it's true that we will have the default `div` with no extra stuff but that means that bootstrap will not handle that element and therefore we may expect issues on responsiveness.
 As set by the bootstrap doc _"Containers are the most basic layout element in Bootstrap and are **required when using our default grid system**."_ 
 But for the record since we have a top level container it's not required to nest containers as set in the docs  _" While containers can be nested, most layouts do not require a nested container."_ 
But, it still a best practice because with react each component encapsulates its UI and can be used in other projects,

[More](https://getbootstrap.com/docs/5.0/layout/containers/#how-they-work)

Reply:
I avoid nesting layout elements.

I don't see why we shouldn't use regular <div> because we are using bootstrap.

I am confident using a <container> in this particular place will mess up the layout responsiveness.